<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextMenu</class>
 <widget class="QWidget" name="TextMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text menu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type your text here:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="Textinput">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QPushButton" name="uploadButton">
     <property name="text">
      <string>UPLOAD TO HELMET</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QPushButton" name="pushReturn">
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="1">
    <widget class="QRadioButton" name="loopButton">
     <property name="text">
      <string>REPEAT</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="scrollButton">
     <property name="text">
      <string>MARQUEE MODE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="animationControl">
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>UPDATE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>RUN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="periodSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>PERIOD[sec]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="navigationLayout">
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="text">
        <string>PREV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="navSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>NEXT</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="pageNmbr">
     <property name="text">
      <string>#/#</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
